templates/real-time/index.html

```diff
@@ -0,0 +1,219 @@
+    <div class="col">
+      <span class="p-text--small-caps">Video</span>
+      <h2>What is Real-time Ubuntu?</h2>
+    </div>
+    <div class="col">
+      <div class="u-embedded-media u-no-margin--bottom">
+        <iframe title="YouTube video player: What is Real-time Ubuntu?" class="u-embedded-media__element" src="https://www.youtube.com/embed/fxoWWfEWbIE?si=3Gi3EmZEhoQekrun" allowfullscreen=""></iframe>
+      </div>
+      <p>Canonical and Intel collaborate to deliver real-time Ubuntu on 12th Gen Intel&reg; Core&trade; processors with TSN and TCC Support. Explore how real-time Ubuntu benefits the IoT customers and market as a whole.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>Getting started</h2>
+    </div>
+    <div class="col">
+      <p>The real-time kernel is available via <a href="/pro">Ubuntu Pro</a>, Canonical’s enterprise security and compliance subscription. Ubuntu Pro is free for personal and small scale commercial use in up to 5 machines.</p>
+      <p>With an Ubuntu Pro subscription, launching the real-time kernel is as easy as:</p>
+      <pre><code>pro attach &lt;token&gt;
+pro enable realtime-kernel</code></pre>
+      <a class="p-button" href="https://ubuntu.com/pro">Discover Ubuntu Pro</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>Run Real-time Ubuntu on Intel hardware</h2>
+    </div>
+    <div class="col">
+      <p>Optimised Real-time Ubuntu is available on 12th Gen Intel&reg; Core&trade; processors. Coupled with Intel&reg; Time Coordinated Computing (Intel&reg; TCC) and Time Sensitive Networking (IEEE TSN), the real-time kernel delivers industrial-grade performance to meet stringent latency requirements:</p>
+      <p>Intel-optimised Real-time Ubuntu is available on Intel SoCs via Ubuntu Pro:</p>
+      <pre><code>pro attach &lt;token&gt;
+pro enable realtime-kernel --variant intel-iotg</code></pre>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>Learn more about Real-time Ubuntu</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-2">
+              <h3 class="p-text--small-caps">Webinars</h3>
+            </div>
+            <div class="col-4">
+              <ul class="p-list--divided u-no-margin--bottom">
+                <li class="p-list__item"><a href="https://www.brighttalk.com/webcast/6793/535454">An introduction to real-time Linux</a></li>
+                <li class="p-list__item"><a href="https://www.brighttalk.com/webcast/6793/579183?utm_source=Canonical&utm_medium=brighttalk&utm_campaign=579183">How real-time Linux accelerates industrial transformation</a></li>
```

Comment:
Left a question in the doc about this one, not sure if we should be including the utm or not (not a blocker though)